contents/blog/ab-testing-mistakes.md

```diff
@@ -0,0 +1,99 @@
+
+To avoid this problem, you should first test your experiment with a small rollout (e.g., 5% of users) for a few days. Once you're confident everything works correctly, you can start the experiment with the remaining users.
+
+## 4. Including unaffected users in your experiment
+
+Another common mistake in A/B testing is counting users in your experiment who aren't actually affected by the change you're testing. This can happen if you're not careful in how and when you assign users to the control and test groups. It dilutes your experiment results, making it harder to determine the impact of your changes. 
+
+To see how, let's look at an example: Say you're testing a new feature in your app that rewards users for completing a certain action. You mistakenly included users who have already completed the action in the experiment. Since they are not affected by the change, any metrics related to this action do not change, and thus the results for this experiment may not show a statistically significant change.
+
+To avoid this mistake, make sure to first filter out ineligible users in your code before including them in your experiment. Below is an example of how to do this:
+
+```javascript
+// Incorrect. Will include unaffected users
+function showNewChanges(user) {
+  if (user.getExperimentVariantGroup() === 'control') {
+    return false;
+  }
+
+  if (user.hasCompletedAction) {
+    return false
+  }
+
+  // other checks
+  ...
+
+  return true
+}
+```
+
+```javascript
+// Correct. Will exclude unaffected users
+function showNewChanges(user) {
+
+  if (user.hasCompletedAction) {
+    return false
+  }
+
+  // other checks
+  ...
+
+  if (user.getExperimentVariantGroup() === 'control') {
+    return false;
+  }
+
+  return true
+}
+```
+
+## 5. Not determining experiment duration in advance
+
+Not checking your statistical power and not predetermining the duration of your experiment can lead to problems. In the scenario where you don't have enough statistical power (i.e., not enough users to obtain a significant result), you'll end up potentially wasting time waiting for results. Alternatively, you may even fall victim to the "[peeking problem](https://gopractice.io/data/peeking-problem/)": when you check the intermediate results for statistical significance and make decisions based on them. Without defining your experiment run time, you cannot differentiate between intermediate and final results.
+
+Thus the solution is to use an A/B test running time calculator to determine if you have the required statistical power to run your experiment and for how long you should run your experiment.
+
+![Setting up a new experiment in PostHog includes a recommended running time calculator](../images/blog/ab-testing-mistakes/minimum-acceptable-improvement.png)
+<Caption>Setting up a new experiment in PostHog includes a recommended running time calculator</Caption>
```

Comment:
Ideas for outro:
- What do we do at PostHog to prevent these mistakes?
- What features of PostHog help prevent these mistakes?
- Why you should care about preventing mistakes? What can happen if you have sloppy A/B testing practices?